nocodb/nocodb
# Fixed #5765 form view do not extent text area

## Change Summary

This change ensures that the nc-form-after-submit-msg text area no longer allows the text to overflow outside the container. Users can now enter and view large messages within the text area, with the ability to scroll through the content when necessary fixes #5765

## Change type

- [x] fix: form view do not extend text area


## Test/ Verification

To test and verify the fix, follow these steps:
- Compile and build the project to ensure you have the latest changes.
- Navigate to the area where the nc-form-after-submit-msg text area is present.
- Enter a large message that would normally cause the text to overflow.
- Verify that the text area now displays a scrollbar and the text does not extend beyond its boundaries.
- Scroll within the text area to confirm that you can view the entire content of the message.
- Repeat the test with different message lengths to ensure the fix handles various scenarios.


## Additional information / screenshots:

![Screenshot (246)](https://github.com/nocodb/nocodb/assets/139473679/34062503-ba3e-4c46-8fec-d916a1bec9ac)
